docs/troubleshooting.md

```diff
@@ -109,20 +118,26 @@ If everything seems healthy, and still no packages show up in the UI, it could b
 
 ## OperatorSource `MESSAGE` reads unknown error (status 404)
 
-Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong end point being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
+Some of the common reasons we have encountered that caused this `MESSAGE` are quay.io being unavailable or the wrong endpoint being listed in the OperatorSource. If the steps listed [here](#operatorSource-failing-to-download-from-datastore) does not help debug the problem, please reach out to one of the forums listed in [Where to go for help](#where-to-go-for-help).   
+
+## Operator/s in an OperatorSource fail to show up in the UI
+
+If operators in a particular OperatorSource fail to show up in the UI, it could be because those operators had parsing errors and were ignored by the registry pod. To inspect the corresponding registry pod logs, first identify the name of the registry pod for the OperatorSource with `oc get pods -n openshift-marketplace` (the name of the pod should be of the format `<operator-source-name>-<random-characters>`). Get the logs for the pod with `oc logs <identified-pod-name> -n openshift-marketplace`.
+
+If there are private app-registry repositories in your namespace, not specifying the authenticationToken in the OperatorSource CR will result in them not being listed. Please follow the steps [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md) for adding the token to the CR.
 
 ## CatalogSourceConfig stuck in Configuring phase
 
-If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. If you have private app-resgistries, please follow the steps listed [here](https://github.com/operator-framework/operator-marketplace/blob/master/docs/how-to-authenticate-private-repositories.md).
+If you are trying to [install an operator from the cli](https://github.com/operator-framework/operator-marketplace#installing-an-operator-using-marketplace), the packages listed in the CatalogSourceConfig must have valid package names. For guidelines on what contitutes as a valid package name, please see the example [here](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md#create-your-quay-app-registry-repository). Note that the `packageName` of your application should be unique, i.e it cannot have the same `packageName` as an existing application in any other OperatorSource in the cluster.  
```

Comment:
@aravindhp I noticed @awgreene is adding a section about packageNames in a follow up PR. @awgreene please note this addition about packageNames in this PR.